Brokkr/src/main/cpp/commands/SetArmPostitionForDistance.cpp

```diff
@@ -18,57 +27,74 @@ namespace{
     return angle = atan(height/distance);
   }
 
-  double GetNodeDistance(const NodeLevel& NodeLevel, double distance)
+    const double kHybridHeight = 0;
+    const double kMiddleNodeHeight = 34;
+    const double kUpperNodeHeight = 46;
+
+  double GetNodeDistance(const NodeLevel& NodeLevel, double AngleTargetOffset)
   {
+    double distance = 0;
     switch (NodeLevel)
     {
     case NodeLevel::HybridLevel:
-      return GetDistance(0, distance);
+      distance = GetCameraDistance(kHybridHeight, AngleTargetOffset);
```

Comment:
`GetCameraDistance` is called for both GetNodeDistance and GetNodeAngle. 

Consider optimizing to call `GetCameraDistance` only once.

We should make sure that the math is correct _BEFORE_ optimizing.